wormhole-connect/src/sdk/gasEstimates.ts

```diff
@@ -55,12 +57,15 @@ const estimateGasFee = async (
         undefined,
       );
     } else {
-      const parsedNativeAmt = toNativeToken
-        ? utils.parseUnits(toNativeToken, decimals).toString()
-        : '0';
+      const parsedNativeAmt = utils
```

Comment:
Could these changes and those in line 108 be extracted outside the conditionals? It seems to be the same code for both Sui and Eth cases